### PR TITLE
LiveData integration

### PR DESCRIPTION
**Pull request by SrMonedero**

### CODE REVIEW

File: app/src/main/java/com/theleftbit/gifwallet/presentation/GifListViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+class GifListViewModel: ViewModel() {
+
+    val urls = MutableLiveData<List<String>>()
+
+    init {
+        val client = GPHApiClient("API-KEY")
+        client.trending(MediaType.gif, null, null, null) { result, _ ->
+            result?.data.let {
+                urls.postValue(ArrayList<String>().apply {
+                    result.data.forEach {
```
Comment (ignasi) [inline app/src/main/java/com/theleftbit/gifwallet/presentation/GifListViewModel.kt]: Use `it` here, or give it a name to the inner reference.

File: app/src/main/java/com/theleftbit/gifwallet/presentation/GifListViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                        add(it.images.original.gifUrl)
```
Comment (ignasi) [inline app/src/main/java/com/theleftbit/gifwallet/presentation/GifListViewModel.kt]: Isn't there any `addAll` method?

File: build.gradle [hunk 3]
```diff
@@ -1,21 +1,16 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.appcompat_version = '27.1.1'
```
Comment (ignasi) [inline build.gradle:4]: Why moving this versions? I have always seen it in the 'global' gradle scope

**Review by ignasi — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 3 above)
Comment (SrMonedero) [reply build.gradle:4]: It's been always weird for me to have these versions in a file where are never used while they are only used in another file

**Review by ignasi — APPROVED** [review]

**Pull request merged**